episodes/09-formative-assessment.md

```diff
@@ -56,6 +87,27 @@ the topic you are teaching.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+
+[Multiple choice question (MCQ)](https://carpentries.github.io/instructor-training/02-practice-learning/#using-formative-assessment-to-identify-misconceptions) exercises are types of a formative assessment that can help you target anticipated misconceptions. When designed carrefully, each incorrect answer in a MCQ will have diagnostic power and provide valuable insight into how a mental model is broken. For example, suppose we are teaching children multi-digit addition. An example of a well-designed MCQ ([borrowed from The Carpentries Instructor Training](https://carpentries.github.io/instructor-training/02-practice-learning/#using-formative-assessment-to-identify-misconceptions)) in this case could be: 
+
+Q: What is 27 + 15?
+
+a) 42
+b) 32
+c) 312
+d) 33
```

Comment:
```suggestion
> Q: What is 27 + 15?
> 
> a) 42
> b) 32
> c) 312
> d) 33
```
Can this still be in a block quote?  Make it stand out a little.

Reply:
Yes! I was not sure how that is done in the new lesson infrastructure and I was just rushing to submit the first draft. Thank you @sstevens2.